arvados/shell/package/install.sls

```diff
@@ -33,6 +33,12 @@ arvados-shell-package-install-pkg-installed:
       {%- endfor %}
 
 arvados-shell-package-install-gems-deps-pkg-installed:
+  {%- if grains.os == 'OEL' %}
+  cmd.run:
+    - name: "yum-config-manager --enable ol7_optional_latest"
+    - require_in:
+      - pkg: arvados-shell-package-install-gems-deps-pkg-installed
+  {%- endif %}
```

Comment:
This is needed to install the `ruby-devel` package.

Not particularly enamoured with this block.  The repo is included in the distro by default but needs to be enabled with this command -- there's probably a better way to do this than `cmd.run`.  Also, this repo is Oracle Linux 7 specific so perhaps should be providing it in `osfingermap.yaml`, so that the Oracle Linux 8 one can be added there as well.

Reply:
OK, I've just established that Oracle Linux 8 has the repo enabled by default so installs the `ruby-devel` package with out this -- I've updated to check for `osfinger` instead.

The use of `cmd.run` is still something that might be improved, though.